sdk/batch/arm-batch/package.json

```
@@ -20,18 +20,26 @@
   "module": "./esm/batchManagementClient.js",
   "types": "./esm/batchManagementClient.d.ts",
   "devDependencies": {
-    "typescript": "^3.1.1",
-    "rollup": "^0.66.2",
-    "rollup-plugin-node-resolve": "^3.4.0",
+    "@types/chai": "^4.1.6",
+    "@types/jssha": "^2.0.0",
+    "@types/mocha": "^7.0.2",
+    "chai": "^4.2.0",
+    "dotenv": "^8.2.0",
+    "mocha": "^7.1.1",
+    "rollup": "^1.16.3",
+    "rollup-plugin-node-resolve": "^5.2.0",
+    "rollup-plugin-sourcemaps": "^0.4.2",
+    "ts-mocha": "^7.0.0",
```

If you agree with the change to the test script, these deps will be needed instead.
```suggestion
     "esm": "^3.2.25",
    "ts-node": "^8.3.0",
```